README.md

```diff
@@ -28,6 +28,27 @@ Token username as *RHT_REG_SVCUSER* (has a "|" character in the name)
 
 Password as *RHT_REG_SVCPASS*
 
+### Possible runner disk space issue
+These build are full operating systems so can be very big builds depending on the amount of software installed.  Valentin rerported this issue with a different bootc based GitHub Action build:
```

Comment:
The first sentence is hard for me to brain parse.

Reply:
Agreed, that is not at all clear, I rewrote this block to try to simplify and updated the GitHub callouts to work right.